### PR TITLE
Updated documentation and added sealed secrets lab

### PR DESCRIPTION
**Pull request by CasLubbers**

### CODE REVIEW

File: docs/get-started/installation/scaleway.md [hunk 1]
```diff
@@ -79,14 +79,14 @@ cluster:
   provider: scaleway
   domainSuffix: your-sub-domain.example.com
 otomi:
-  hasExternalDNS: true
```
Comment (j-zimnowoda) [inline docs/get-started/installation/scaleway.md:82]: is should be true if you want to use the `domainSuffix`

File: docs/get-started/labs/lab-34.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
+---
+slug: lab-34
+title: Use Sealed Secrets
+sidebar_label: Create RabbitMQ cluster
```
Comment (j-zimnowoda) [inline docs/get-started/labs/lab-34.md:4]: label should match title

File: docs/get-started/labs/lab-34.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+---
+
+In this lab we will create sealed secrets and see how they can be publicly stored in a git repository.
```
Comment (j-zimnowoda) [inline docs/get-started/labs/lab-34.md:7]: how they can be publicly stored in a git repository. => how to securely sensitive information in git repository

File: docs/get-started/labs/lab-34.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
+---
```
Comment (j-zimnowoda) [inline docs/get-started/labs/lab-34.md:1]: This is very good lab. Next time consider presenting a feature from a regular team perspective not team-admin.

**Review by j-zimnowoda — CHANGES_REQUESTED** [review]
Thanks for your contribution. Just few remarks.

File: docs/get-started/labs/lab-34.md (hunk 4 above)
Comment (CasLubbers) [reply docs/get-started/labs/lab-34.md:1]: Ah yeah good one, I think I had some issues with enabling sealed-secrets for a team. Just posted it in the slack channel.

Comment (CasLubbers) [reply docs/get-started/labs/lab-34.md:1]: Updated it with team-demo

**Pull request merged**